spec.html

```diff
@@ -8450,6 +8450,17 @@ <h1>Module Namespace Exotic Objects</h1>
               A List containing the String values of the exported names exposed as own properties of this object. The list is ordered as if an Array of those String values had been sorted using `Array.prototype.sort` using *undefined* as _comparefn_.
             </td>
           </tr>
+          <tr>
+            <td>
+              [[HostDefined]]
+            </td>
+            <td>
+              Any, default value is *undefined*.
```

Comment:
The typical term used here is `Any ECMAScript language type`, i believe

Reply:
The wording here was already used in eg Realm.

Reply:
Fair enough; we can update them all separately.

Reply:
@jmdyck thanks for looking into this further, it's good to ensure these conventions. Do you think sticking with one of the more explicit options would be better then? Although as @ljharb mentions perhaps this could be a separate editorial PR too.

Reply:
I think what you've got is fine for this PR, and consistification should be left to a separate PR. (= agree with @ljharb, sounds like)